src/sentry/eventstream/snuba.py

```diff
@@ -32,6 +32,8 @@
 if TYPE_CHECKING:
     from sentry.eventstore.models import Event
 
+DATETIME_FORMAT = "%Y-%m-%dT%H:%M:%S.%fZ"
```

Comment:
i am sure that this constant is already defined in `sentry.utils.json` somewhere?

Reply:
There was only a copy in synchronized.py. I moved it here since it wasn't used there.

Reply:
i mean, can you use this instead? https://github.com/getsentry/sentry/blob/236822aa884450b3a2e6ca4a0b2aa71fb2b20cdf/src/sentry/utils/json.py#L26-L27

this is the function we serialize datetimes with today

Reply:
Oh I missed that one, will do